File: app_watcher/src/main/java/com/tongchen/twatcher/mzitu/presenter/IMZiTuCategoryPresenter.java
package com.tongchen.twatcher.mzitu.presenter;

import com.tongchen.twatcher.base.presenter.IMVPPresenter;
import com.tongchen.twatcher.mzitu.view.IMZiTuCategoryView;

/**
 * Created by TongChen at 11:07 on 2018/12/28.
 * <p>
 * Description: 妹子图按分类获取封面列表
 */
public interface IMZiTuCategoryPresenter extends IMVPPresenter<IMZiTuCategoryView> {

    void getMZiTuCoverListByCategory(String category, boolean pullToRefresh);
}

File: app_watcher/src/main/java/com/tongchen/twatcher/mzitu/presenter/IMZiTuCategoryPresenter.kt
package com.tongchen.twatcher.mzitu.presenter

import com.tongchen.twatcher.base.presenter.IMVPPresenter
import com.tongchen.twatcher.mzitu.view.IMZiTuCategoryView

/**
 * @author TongChen
 * @date 2019/11/28  14:49
 * <p>
 * Desc: 妹子图按分类获取封面列表
 */
interface IMZiTuCategoryPresenter : IMVPPresenter<IMZiTuCategoryView> {

    fun getMZiTuCoverListByCategory(category: String?, pullToRefresh: Boolean)
}